doc/classes/ProjectSettings.xml

```diff
@@ -2968,23 +2972,28 @@
 			The number of entries in the sampler descriptors heap the Direct3D 12 rendering driver uses each frame, used for most rendering operations.
 			Depending on the complexity of scenes, this value may be lowered or may need to be raised.
 		</member>
-		<member name="rendering/rendering_device/driver" type="String" setter="" getter="">
+		<member name="rendering/rendering_device/driver" type="String" setter="" getter="" default="&quot;auto&quot;">
 			Sets the driver to be used by the renderer when using a RenderingDevice-based renderer like the Forward+ or Mobile renderers. This property can't be edited directly. Instead, set the driver using the platform-specific overrides. This can be overridden using the [code]--rendering-driver &lt;driver&gt;[/code] command line argument.
+			Supported values are:
+			- [code]auto[/code], Metal on Apple Silicon Macs and iOS, D3D12 on Windows if engine was compiled w/o Vulkan support, Vulkan in all other cases.
+			- [code]metal[/code], Metal (supported on Apple Silicon Macs and iOS).
+			- [code]vulkan[/code], Vulkan (supported on all desktop and mobile platforms).
+			- [code]d3d12[/code], Direct3D 12 (supported on Windows).
 			[b]Note:[/b] The actual rendering driver may be automatically changed by the engine as a result of a fallback, or a user-specified command line argument. To get the actual rendering driver that is used at runtime, use [method RenderingServer.get_current_rendering_driver_name] instead of reading this project setting's value.
 		</member>
-		<member name="rendering/rendering_device/driver.android" type="String" setter="" getter="">
+		<member name="rendering/rendering_device/driver.android" type="String" setter="" getter="" default="&quot;auto&quot;">
 			Android override for [member rendering/rendering_device/driver].
```

Comment:
It's a bit redundant, but I'd add details about what `auto` does for each platform override, so it's crystal clear.
```suggestion
			Android override for [member rendering/rendering_device/driver].
			The [code]auto[/code] setting is equivalent to [code]vulkan[/code] on this platform.
			[b]Note:[/b] If Vulkan was disabled at compile time, there is no alternative RenderingDevice driver.
```